tests/test_autonames/test_autonames.py

```diff
@@ -10,11 +10,19 @@
 def test_autonames():
     """Here we check that bad migrations do produce warnings."""
     warnings = check_migration_names()
+    warnings_msgs = [i.msg for i in warnings]
```

Comment:
Personally I dislike names like `i`, `j` etc and I use it only to store index values e.g.
 `for i, some_element in enumerate(some_iterable): ...`

Could you update `i` to something more meaningful? For instance:

```suggestion
    warnings_msgs = [warning.msg for warning in warnings]
```